packages/relay-compiler/codegen/persistQuery.js

```diff
@@ -0,0 +1,7 @@
+import md5 from '../util/md5';
```

Comment:
This file probably needs a FB header like all other files.

Reply:
Done